pkg/controller/command/verifiable/command.go

```diff
@@ -76,17 +125,28 @@ type provider interface {
 // Command contains command operations provided by verifiable credential controller.
 type Command struct {
 	verifiableStore *verifiablestore.Store
+	didStore        *didstore.Store
+	kms             legacykms.KMS
+	kResolver       keyResolver
 }
 
 // New returns new verifiable credential controller command instance.
-func New(p provider) (*Command, error) {
+func New(p provider, kms legacykms.KMS, kResolver keyResolver) (*Command, error) {
```

Comment:
kms can be added as part of the [provider](https://github.com/hyperledger/aries-framework-go/pull/1600/files#diff-957d4786eb551ebc07beb3497e685537R122) instead of new param  as it's already available in context https://github.com/hyperledger/aries-framework-go/pull/1600/files#diff-3c862eb54a8e0e161b534e0c67e5379eR190.

Reply:
done